CI fails for experimental tests: TypeError: argument should be a str or an os.PathLike object where __fspath__ returns a str, not 'NoneType'

CI fails for experimental tests: https://github.com/huggingface/trl/actions/runs/21428384040/job/61702068799
> TypeError: argument should be a str or an os.PathLike object where __fspath__ returns a str, not 'NoneType'

```python
FAILED tests/experimental/test_judges.py::TestJudges::test_pair_rm_judge - TypeError: argument should be a str or an os.PathLike object where __fspath__ returns a str, not 'NoneType'
FAILED tests/experimental/test_judges.py::TestJudges::test_pair_rm_judge_return_scores - TypeError: argument should be a str or an os.PathLike object where __fspath__ returns a str, not 'NoneType'
```

Traceback:
```python
  ________________________ TestJudges.test_pair_rm_judge _________________________
  [gw3] linux -- Python 3.13.11 /__w/trl/trl/.venv/bin/python3
  
  self = <tests.experimental.test_judges.TestJudges object at 0x7fa3c3808b00>
  
      @require_llm_blender
      @pytest.mark.skipif(
          sys.version_info[:3] == (3, 13, 8), reason="Python 3.13.8 has a bug in inspect.BlockFinder (cpython GH-139783)"
      )
      def test_pair_rm_judge(self):
  >       judge = self.load_pair_rm_judge()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^
  
  tests/experimental/test_judges.py:77: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  tests/experimental/test_judges.py:67: in load_pair_rm_judge
      return PairRMJudge()
             ^^^^^^^^^^^^^
  trl/experimental/judges/judges.py:234: in __init__
      self.blender.loadranker("llm-blender/PairRM", device=Accelerator().device)
  .venv/lib/python3.13/site-packages/llm_blender/blender/blender.py:97: in loadranker
      cache_dir = Path(cache_dir)
                  ^^^^^^^^^^^^^^^
  /__w/_tool/Python/3.13.11/x64/lib/python3.13/pathlib/_local.py:503: in __init__
      super().__init__(*args)
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  
  self = <[AttributeError("'pathlib._local.PosixPath' object has no attribute '_raw_paths'") raised in repr()] PosixPath object at 0x7fa2d1334510>
  args = (None,), paths = [], arg = None, path = None
  
      def __init__(self, *args):
          paths = []
          for arg in args:
              if isinstance(arg, PurePath):
                  if arg.parser is not self.parser:
                      # GH-103631: Convert separators for backwards compatibility.
                      paths.append(arg.as_posix())
                  else:
                      paths.extend(arg._raw_paths)
              else:
                  try:
                      path = os.fspath(arg)
                  except TypeError:
                      path = arg
                  if not isinstance(path, str):
  >                   raise TypeError(
                          "argument should be a str or an os.PathLike "
                          "object where __fspath__ returns a str, "
                          f"not {type(path).__name__!r}")
  E                   TypeError: argument should be a str or an os.PathLike object where __fspath__ returns a str, not 'NoneType'
  
  /__w/_tool/Python/3.13.11/x64/lib/python3.13/pathlib/_local.py:132: TypeError
```

--- a/tests/experimental/test_judges.py
+++ b/tests/experimental/test_judges.py
@@ -75,9 +75,10 @@ class TestJudges(TrlTestCase):
     @pytest.mark.skipif(
         sys.version_info[:3] == (3, 13, 8), reason="Python 3.13.8 has a bug in inspect.BlockFinder (cpython GH-139783)"
     )
-    @pytest.mark.skipif(
+    @pytest.mark.xfail(
         Version(transformers.__version__) >= Version("5.0.0"),
-        reason="llm-blender is not compatible with transformers >= 5.0.0 (GH-4918)",
+        reason="Known incompatibility between llm-blender and transformers >= 5.0.0 (GH-4918)",
+        strict=True,
     )
     def test_pair_rm_judge(self):
         judge = self.load_pair_rm_judge()
@@ -91,9 +92,10 @@ class TestJudges(TrlTestCase):
     @pytest.mark.skipif(
         sys.version_info[:3] == (3, 13, 8), reason="Python 3.13.8 has a bug in inspect.BlockFinder (cpython GH-139783)"
     )
-    @pytest.mark.skipif(
+    @pytest.mark.xfail(
         Version(transformers.__version__) >= Version("5.0.0"),
-        reason="llm-blender is not compatible with transformers >= 5.0.0 (GH-4918)",
+        reason="Known incompatibility between llm-blender and transformers >= 5.0.0 (GH-4918)",
+        strict=True,
     )
     def test_pair_rm_judge_return_scores(self):
         judge = self.load_pair_rm_judge()

